docs/content/posts/2023-04-12-howto-migrate-to-datastream.md

```diff
@@ -2,29 +2,39 @@
 title:  "Howto migrate a real-life batch pipeline from the DataSet API to the DataStream API"
 date: "2023-04-12T08:00:00.000Z"
 authors:
+
 - echauchot:
   name: "Etienne Chauchot"
   twitter: "echauchot"
-aliases:
-- /news/2023/04/12/2023-04-12-howto-migrate-to-datastream.html
+  aliases:
+- /2023/04/12/2023-04-12-howto-migrate-to-datastream.html
```

Comment:
we shouldnt need an alias; this is only required for older blog posts.

Reply:
fixed